semantic_conventions/logs/log-general.yaml

```
@@ -0,0 +1,17 @@
+groups:
+  - id: log.record
+    prefix: log.record
+    brief: >
+      The attributes described in this section are rather generic. They may be used in any Log Record they apply to.
+    attributes:
+      - id: id
+        type: string
+        requirement_level: recommended
+        brief: >
+          A unique identifier for the Log Record.
+        note: >
+          MUST be an [Universally Unique Lexicographically Sortable Identifier (ULID)](https://github.com/ulid/spec).
+          If a value is provided other log records, with the same value, will be considered to be duplicates and can be removed safely.
```

```suggestion
          If an id is provided, other log records with the same id will be considered to be duplicates and can be removed safely.
```
Nit: I feel using `id` makes things more clear than just `value`

---

👍